src/components/Skeleton/SkeletonConstants.ts

```diff
@@ -1,32 +1,35 @@
 // Fixed names
-export const SKELETON_CIRCLE_TYPE = "circle";
-export const SKELETON_TEXT_TYPE = "text";
-export const SKELETON_RECTANGLE_TYPE = "rectangle";
 export const SKELETON_CUSTOM_SIZE = "custom";
 
-export const SKELETON_TYPES = {
-  CIRCLE: SKELETON_CIRCLE_TYPE,
-  RECTANGLE: SKELETON_RECTANGLE_TYPE,
-  TEXT: SKELETON_TEXT_TYPE
-};
+export enum SkeletonType {
+  CIRCLE = "circle",
+  RECTANGLE = "rectangle",
+  TEXT = "text"
+}
+
+export enum TextSkeletonSize {
+  H1 = "h1",
+  H2 = "h2",
+  H3 = "h3",
+  H4 = "h4",
+  H5 = "h5",
+  H6 = "h6",
+  P = "p",
+  SMALL = "small"
+}
 
-export const SKELETON_ALLOWED_TYPES = Object.values(SKELETON_TYPES);
+export enum ShapeSkeletonSize {
```

Comment:
why enum is better here than just const? Also the name is rather strange

Reply:
The reason I had to change it was I had issues using the skeleton with typescript in the previous version (when I used the skeleton for the dialog). So I must do change there for typescript users.
There is 2 types here: shape type (circle, rectangle and text) and size (there is a enum for text skeleton and only one valid size or the other skeleton types). because of this weird structure which I can't break, I create 2 enums: skeleton type and skeleton text size and also create a const for the size of circle or rectangle.
I admit the name is a bit weird, If you have a better idea we can change